pagina de Disciplinas

MarcusMoraesOliveira opened the pull request:
# [Grupo 2] Manutenção de Disciplinas

* operações de inserir,deletar, e alterar
* exceções

MarcusMoraesOliveira committed:
pagina de Disciplinas

Thileps commented:
Tá faltando implementar o setResizable no ErroMain. Deixa o fundo do título da coluna da tabela branco.

MarcusMoraesOliveira committed:
Merge branch 'dev' into feature/diario/disciplinas

isabellaCE committed:
Função voltar para tela de transição

isabellaCE committed:
Conserto na estilização da tabela e colocação do setResizable() na classe ErroMain

MarcusMoraesOliveira committed:
deletando arquivo que gerou aqui

Thileps commented:
Tá sobrando um espaço vazio nos lado do header q n devia existir; stage.setWidth(1280) no main deve corrigir.

isabellaCE committed:
Espaço vazio nos lados do header resolvido
